ssl_verification_test.go

```diff
@@ -0,0 +1,60 @@
+package cloudflare
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListSSLVerification(t *testing.T) {
+	setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "GET", r.Method, "Expected method 'GET', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprint(w, `{
+			"result": [
+				{
+					"certificate_status": "pending_validation",
+					"cert_pack_uuid": "4449b368-d10f-4a73-9d27-f3f2752443df",
+					"validation_method": "http",
+					"validation_type": "dv",
+					"verification_info": {
+						"http_url": "http:\/\/example.com\/.well-known\/pki-validation\/ca3-0fdf5b3dc4d0428fa81ac381a8c4e333.txt",
+						"http_body": "ca3-ff2d513629514e3b90675a982f9f0644"
+					},
+					"hostname": "example.com"
```

Comment:
This response object seems to differ what is coming back from the API and is documented. Here is what I'm getting from the API with an active verification using some anonymised fields:

```json
{
  "certificate_status": "active",
  "signature": "ECDSAWithSHA256",
  "brand_check": true,
  "verification_info": {
    "record_name": "abc123.example.com",
    "record_target": "72AA6FC86BB81C0BA35B6DBB66481234.comodoca.com"
  },
  "verification_status": null,
  "verification_type": "cname"
}
```

The differences look to be:

- `brand_check` is missing and a boolean 
- `verification_info` object has different key values (`record` instead of `http`) which will make a difference in your `ZoneSSLVerification` struct. Perhaps we need to account for both of these structs separately and allow `VerificationInfo` to accept either?
- `cert_pack_uuid` is missing from my response and the documentation.

I'm unsure why there are differences but I've tested this response on an Enterprise and Personal paid plans with no differences to the response. Are you on an older SSL product or something along those lines that we may need to account for here?